### ISSUE TITLE
About the configuration file of DeepLabV2

### ISSUE THREAD

**Issue by dongjunhwang** [zhaozhengChen/ReCAM#23]
Hello, @zhaozhengChen. I really appreciate you sharing this excellent code.

Could you please let me know about the configuration file of DeepLabV2?

Your paper reports only the crop size / iterations / weight decay / batch size and data augmentation techniques.
But many other WSSS papers also report the multi-scale testing during inference.
Is this paper use multi-scale testing following the IRN and AdvCAM?

Also, do you just totally follow the default setting and code in [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch)?
If you change the part of the deeplab-pytorch codes, then can you share your own code?

Thanks.

**Comment by zhaozhengChen** [comment]
Hi, thanks for your interest in our work!
Yes, we totally follow the default setting (the config file at this [link](https://github.com/kazuto1011/deeplab-pytorch/blob/master/configs/voc12.yaml) )and code in [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch).

**Comment by dongjunhwang** [comment]
Thank you for your answer.

By the way, we couldn't get the reported mIoU score (**68.50** in VOC) in the semantic segmentation network (deeplabv2) trained by the pseudo label which has **70.81** mIoU at VOC training set. (and we use ImageNet pretrained model to train deeplabv2.)

we trained deeplabv2 five times, but we got 68.02, 68.17, 68.25, 67.84, and 67.92 mIoU scores.
How many trials did you try to get this score?

**Comment by zhaozhengChen** [comment]
We reported the average score of three runs (68.2, 68.5, 68.7). 
It is possible that there are slight differences in the pseudo labels used, which could contribute to the difference.

**Comment by dongjunhwang** [comment]
Thank you for your patient reply.

To someone who suffers from this issue, I found the [similar issue](https://github.com/linyq2117/CLIP-ES/issues/7) which points out the consistency of the segmentation performance using [deeplab-pytorch ](https://github.com/kazuto1011/deeplab-pytorch) code.
